src/components/core/Header.vue

```diff
@@ -30,20 +35,22 @@ q-toolbar.row.q-py-sm.q-px-md.bg-grey-1
 
     .col.row.justify-center
       search-bar.toolbar-select.q-mr-md(
-        size='35',
-        v-if='$q.screen.gt.xs && $q.screen.lt.md'
+        size='80',
+        v-if='$q.screen.gt.xs && $q.screen.lt.lg'
       )
 
   .col.row.justify-center
     search-bar.toolbar-select.q-mr-md(size='100', v-if='$q.screen.gt.md')
 
   .q-mr-md.q-gutter-x-md.row.items-center.no-wrap(v-if='$q.screen.gt.md')
-    q-btn(
+    q-btn.btn-left(
       flat,
       icon='grid_view',
       label='Discover',
       dense,
-      :to='{ name: "buy" }'
+      :to='{ name: "buy" }',
+      v-bind:class='{ white: !clicked, blue: clicked }',
+      v-on:click='clicked = !clicked'
```

Comment:
You can also just do @click

Reply:
My apologies Dewlap, just my first time really using Vue. So should I just remove the the v-bind and the v-on?. Are there any more changes apart from that?

Reply:
https://www.vuemastery.com/pdf/Vue-Essentials-Cheat-Sheet.pdf

Have a look there. It's just convenient shortcuts. Try to see where we have used the code before and try to keep to one convention